components/atoms/a-back-to-top.vue

```diff
@@ -11,7 +11,6 @@
         icon="chevron_down"
         icon-size="20px"
         icon-color="white"
-        view-box="0 -4 14 14"
```

Comment:
I removed `view-box` because it is not needed if Storefront UI icon is used.

Reply:
ok